src/languages/en.js

```diff
@@ -588,12 +588,14 @@ export default {
         },
         invite: {
             invitePeople: 'Invite People',
-            invitePeoplePrompt: 'Invite a colleague to your workspace.',
+            invitePeoplePrompt: 'Invite colleagues to your workspace.',
             personalMessagePrompt: 'Add a Personal Message (Optional)',
-            enterEmailOrPhone: 'Email or Phone',
+            enterEmailOrPhone: 'Emails or Phones',
+            EmailOrPhonePlaceholder: 'Enter comma-separated emails or phones',
             pleaseEnterValidLogin: 'Please ensure the email or phone number is valid (e.g. +15005550006).',
             pleaseEnterUniqueLogin: 'That user is already a member of this workspace.',
             genericFailureMessage: 'An error occurred inviting the user to the workspace, please try again.',
+            systemUserError: 'Please ensure email or phone number is not a system User.',
```

Comment:
Coming back from the linked issue, the copy we landed on here is:

> Sorry, you cannot invite chronos@expensify.com to a workspace.

Using whichever invalid email was found first.